### ISSUE TITLE
Informations missing (N/A)

### ISSUE THREAD

**Issue by Scentle5S** [ukdtom/plex2csv.bundle#140]
I have a large library and I want to know easily which files are using a specific video codec (x265). So I export the library at level 4 which is supposed to show the video codec. But the vast majority of my files show N/A data (see attachment), even though all the information is here in the details of each media when looked from Plex. Note that the N/A lines are mixed up : there are both x264 and x265 files in those.

Did I do something wrong or is it a bug ?

![issue](https://cloud.githubusercontent.com/assets/9402790/21075728/48f0d120-bf19-11e6-9ed6-c548075c28b5.png)

**Comment by ukdtom** [comment]
Picture doesn't show :-(

**Comment by Scentle5S** [comment]
Hmm, it shows perfectly well with me...

**Comment by ukdtom** [comment]
Try upload again

**Comment by Scentle5S** [comment]
![issue](https://cloud.githubusercontent.com/assets/9402790/21075933/228f3eee-bf1f-11e6-844e-8eeedd5fc7d1.png)

**Comment by Scentle5S** [comment]
I get it : it doesn't show on chrome but does on safari. Must be due to my hard core conversion of the unsupported tiff file into png by renaming it ><

**Comment by ukdtom** [comment]
Then zip the darn thing

**Comment by Scentle5S** [comment]
<img width="634" alt="issue" src="https://cloud.githubusercontent.com/assets/9402790/21075958/b2ebf90a-bf1f-11e6-99b5-8767acb9d17a.png">

**Comment by ukdtom** [comment]
Need a zip of Plex2CSV log files

**Comment by Scentle5S** [comment]
Can't find the com.plexapp.plugins.plex2csv.log file... Searched literally everywhere (whole drive search). Actually no logs seem to be generated by my Plex, even though everything is set up in the web app settings. I tried downloading the logs from there but no plex2csv logs either...

**Comment by ukdtom** [comment]
https://github.com/ukdtom/plex2csv.bundle/wiki/Troble-Shooting

**Comment by Scentle5S** [comment]
Yes I went there but I just realized that log files aren't in the Plex folder on MacOS (I'm on Windows usually). Preparing zip file to send to you on the Plex forum ;-)

**Comment by ukdtom** [comment]
According to the logs, everything was fine, and what's in the csv file, is what I could get out of Plex

So try this.....

In the Web Client, go to a media that reports as N/A, select media info, and then Show XML

Post the output here

**Comment by Scentle5S** [comment]
Ok so now I'm lost... Last time I checked I was able to find many files which had info in "media info" but reported as N/A in the csv. But now all the files I checked that report as N/A have indeed missing info in the web client (which makes sense). So I tried refreshing the library again (already did when the problem appeared) and this time all files have info and everything shows fine in the csv -> I've no idea what happened in the first place and why the first refresh didn't fix this then ><

Thanks for the help and sorry for the inconvenience.

I definitely love your plugin ;-)
